test/testcafe/framework/shared/index.js

```diff
@@ -80,3 +83,14 @@ export async function assertRequestMatches(loggedRequest, url, method, body) {
     await t.expect(requestBody).eql(body);
   }
 }
+
+/**
+ * Provides mock responses for common endpoints. Use this export instead of
+ * importing from "testcafe" directly to avoid falling back to dyson mock server
+ * for these requests.
+ */
+export const RequestMock = (...args) => TestCafeRequestMock(...args)
+  .onRequestTo('http://localhost:3000/oauth2/default/v1/interact')
+  .respond(xhrInteract)
+  .onRequestTo('http://localhost:3000/oauth2/default/.well-known/openid-configuration')
+  .respond(xhrWellknownOpenidConfiguration);
```

Comment:
for some reason I thought these were already merged

Reply:
I just saw why they weren't the other branch has an unrelated change that broke the suite against v2. Password toggle selector

Reply:
Or maybe my local version was out of date... smh